src/artifact_choosing.rs

```diff
@@ -1,5 +1,12 @@
-#[cfg(target_os = "windows")]
-static PLATFORM_KEYWORDS: &[&str] = &["win32", "win64", "windows"];
+#[cfg(all(target_os = "windows", target_arch = "x86"))]
```

Comment:
I think we could use `target_pointer_width` instead. In practice it probably does not change much but the intent is a little clearer 🙂 

```suggestion
#[cfg(all(target_os = "windows", target_pointer_width = "32"))]
```

Reply:
`target_arch` is more appropriate. For one, the keywords are meant to correspond to an architecture. For another, just pointer width isn't enough to disambiguate from other 64-bit architectures. For example, the code for `macos` in the same file has architectures for `x86_64` and `aarch64`, which both (I'm assuming) have pointer widths of 64.